### DISCUSSION TITLE
Future of this repo after the merge into the mainstream

### DISCUSSION THREAD

**Discussion by blazewicz** [Jyers/Marlin#1379] [General Discussion]
Hi @Jyers,

now that JyersUI is part of mainstream Marlin after https://github.com/MarlinFirmware/Marlin/pull/22422, I'm curious what would be the function of this repository?

I believe both implementations diverged a bit, mainly because of unification between CrealityUI/MarlinUI/JyersUI, and it might be hard or impossible to make PR's from this repo to original one. What is your plan on further development? A rebase leaving only CI with custom configurations?

BTW I just installed latest mainstream with JyersUI enabled on BTT SKR Mini E3 V2.0 and it works great so far.

Thanks!

**Comment by Jyers** [comment]
@blazewicz 
Further development will definitely continue on this branch. I'll probably have to rebase the branch off of the merged marlin implementation, or even just create a new branch and add back in the newest additions. From there I should be able to steadily PR new features into the main marlin repo without much difficulty.

Glad to hear the marin marlin JyersUI is working well, it's such a big step and a huge relief to finally have it upstream :)

**Reply by gothcha** [reply]
Well done Jacob, at the moment I will not be using the main Marlin branch as I have tailored the menus to my liking from all the knowledge gained from working on the firmware along with you. If Marlin add any features that are appropriate for the V2 I can consider incorporating them. Thank you for all the work that you have put into this project.

**Reply by stephenfeather** [reply]
So I want to implement head (physical, not GIT) changes to go along with my Mutant stuff coming this weekend. (Printer, Laser, Pen)
In Marlin Vanilla it was "semi-easy", although it was all shoved into one screen.  The multi UI approach of this firmware is what I found attractive along with the other "minor" changes that have huge impacts on prints.

Unless I missed it, a document that described requirements and entry points into the new UI would be great to have.
